assets/uijson/grid_creation.ui.json

```diff
@@ -0,0 +1,86 @@
+{
+    "title": "Block Model Creation",
+    "geoh5": "",
+    "run_command": "geoapps.grid_creation.driver",
```

Comment:
Can we rename this `block_model_creation`? Will be more obvious the type of objects we are targeting.